src/visiomode/config.py

```diff
@@ -24,11 +23,12 @@
     "width": 400,
     "height": 800,
     "fullscreen": False,
-    "devices": "devices/",
+    "devices": "devices",
+    "config_path": ".visiomode.json",
 }
 
 
-class Config:
+class Config(object):
```

Comment:
Is it necessary to inherit from `object`? We're not looking to be compatible with python 2, is there an additional reason you prefer to inherit explicitly i may not be aware of?